src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -364,7 +357,7 @@ export class LazyMobXTableStore<T> {
             //rowDownloadData is list of lists, containing all the elements per column.
             //processedRowsDownloadData becomes the transposed of rowDownloadData.
             let processedRowsDownloadData = rowDownloadData[0].map(function(
-                row: string,
+                _row: string,
```

Comment:
VS Code suggested prefixing unused variables with an underscore

Reply:
Let's not do that. I guess that's vscode's default naming scheme for unused variables, but that clashes with how we use underscore prefixed variables in our stores. Just leave it as `row`.